metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/ELASTICSEARCH/2.3.3/configuration/elastic-site.xml

```diff
@@ -47,7 +47,15 @@
         <name>path_data</name>
         <value>"/opt/lmm/es_data"</value>
         <description>Path to directory where to store index data allocated for this node. e.g. "/mnt/first", "/mnt/second"</description>
-    </property>    
+    </property>
+    <property>
+        <name>http_cors_enabled</name>
+        <value>"false"</value>
```

Comment:
Assuming that the quotes don't get removed in the .j2 files, can we move the quotes into those, then just leave this as a plain string without quotes? I'm not sure if using the quotes is required or not, but if they are then users might not remember to add them.

If we do want to change that, given that path_data does this even in the Github snippet, I'm sure this occurs more frequently and we may want to create a separate task for cleaning all of them up in one fell swoop.